core/src/main/scala/dagr/core/cmdline/DagrCoreMain.scala

```diff
@@ -142,7 +141,9 @@ class DagrCoreMain(
   @arg(doc = "Set the memory available to dagr.", common = true)
   val memory: Option[String] = None,
   @arg(doc = "Write an execution report to this file, otherwise write to the stdout", common = true)
-  val report: Option[Path] = None
+  val report: Option[Path] = None,
+  @arg(doc = "Provide an top-like interface for tasks with the give delay in seconds. This suppress info logging.")
+  val taskTop: Double = -1
```

Comment:
Definitely needs a better name.  I was thinking about this some and I think I would prefer `interactive`, as this could become a more general switch for an _interactive mode_ that not only displays the top-like console, but perhaps allows other things like dropping to a scala shell etc.